native/acter/src/api/calendar_events.rs

```diff
@@ -348,6 +380,56 @@ impl CalendarEventDraft {
         Ok(())
     }
 
+    pub fn physical_location(
+        &mut self,
+        name: String,
+        description: String,
+        description_html: Option<String>,
+        cooridnates: String,
+        uri: Option<String>,
+    ) -> Result<()> {
```

Comment:
fine for now, but I'd like this to be a builder-pattern in the future rather than additional methods ...